integration_test/cases/browser/set_value_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule Wallaby.Integration.Browser.SetValueTest do
+  use Wallaby.Integration.SessionCase, async: true
+
+  setup %{session: session} do
+    page = visit(session, "forms.html")
+    {:ok, %{page: page}}
+  end
+
+  describe "set_value/3" do
+    test "accepts a query", %{page: page} do
+      page
+      |> send_keys(Query.text_field("Name"), ["Chris", :tab, "c@keathley.io"])
+
+      assert page
+      |> find(Query.text_field("Name"))
+      |> has_value?("Chris")
```

Comment:
This also makes me think that maybe we should have a `value` option for queries (if we don't have that already) so that this could just be: `assert_has(parent, Query.text_field("Name", value: "Chris"))` but that's for another time :)